style.css

```diff
@@ -0,0 +1,138 @@
+body {
+  background: #191765;
+  background-image: url(./images/top-circles.svg);
+  background-repeat: no-repeat;
+  max-width: 375px;
```

Comment:
- [ ] Kindly remove this line of code to make your app fit any mobile screen size. This is causing a gap to appear on the right for any screen bigger than 375px

Reply:
@JuliCarracedo Since it's the mobile version, isn't there supposed to be a max-width?